gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-* Files should be encoded in UTF-8 to support all Unicode characters. Files that include the Unicode byte-order mark (BOM) character are acceptable. Please see the [Unicode FAQ](http://unicode.org/faq/utf_bom.html#BOM) for more information on the BOM character and UTF-8.
-* Zip the files in your feed.
+* Files should be encoded in UTF-8 to support all Unicode characters. Files that include the Unicode byte-order mark (BOM) character are acceptable. See the [Unicode FAQ](http://unicode.org/faq/utf_bom.html#BOM) for more information on the BOM character and UTF-8.
+* All dataset files must be zipped together.
 
 ## Field Definitions
 
 ### agency.txt
 
 File: **Required**
 
-|  Field Name | Type | Required | Details |
+|  Field Name | Type | Required | Description |
 |  ------ | ------ | ------ | ------ |
-|  agency_id | ID | Optional | The **agency_id** field contains an ID that uniquely identifies a transit agency. A transit feed may represent data from more than one agency. This field is optional for transit feeds that only contain data for a single agency. |
-|  agency_name | Text | **Required** | The **agency_name** field contains the full name of the transit agency. Google Maps will display this name. |
-|  agency_url | URL | **Required** | The **agency_url** field contains the URL of the transit agency. |
-|  agency_timezone | Timezone | **Required** | The **agency_timezone** field contains the timezone where the transit agency is located. If multiple agencies are specified in the feed, each must have the same agency_timezone. |
-|  agency_lang | Language code | Optional | The **agency_lang** field contains a language code specifying the primary language used by this transit agency. This setting helps GTFS consumers choose capitalization rules and other language-specific settings for the feed. |
-|  agency_phone | Phone number | Optional | The **agency_phone field** contains a single voice telephone number for the specified agency. This field is a string value that presents the telephone number as typical for the agency's service area. It can and should contain punctuation marks to group the digits of the number. Dialable text (for example, TriMet's "503-238-RIDE") is permitted, but the field must not contain any other descriptive text. |
-|  agency_fare_url | URL | Optional | The **agency_fare_url** specifies the URL of a web page that allows a rider to purchase tickets or other fare instruments for that agency online. |
-|  agency_email | Email | Optional | Contains a single valid email address actively monitored by the agency’s customer service department. This email address will be considered a direct contact point where transit riders can reach a customer service representative at the agency. |
+|  agency_id | ID | **Conditionally Required** | Identifies a transit agency. A dataset may contain data from multiple agencies. This field is required when the dataset contains data for multiple transit agencies, otherwise it is optional. |
```

Comment:
If you wanna be by-the-book consistent, all your "descriptions" you either start by a name, or start by a verb. Here your first one is a verb ("identifies") and the second one is a name ("full name").

Since most of them are already names, you can change agency_id to something like "Identifier which..." or "String that..."

You can also leave it as it. But it's less consistent.